### PR TITLE
Let MMDLoader generate BufferGeometry

### PR DESCRIPTION
**Pull request by takahirox**
`MMDLoader` generated `Geometry`.
I've made it generate `BufferGeometry` now

### CODE REVIEW

File: examples/js/loaders/MMDLoader.js [hunk 1]
```diff
@@ -2605,6 +2635,24 @@ THREE.MMDLoader.prototype.createMesh = function ( model, texturePath, onProgress
...
+
+		geometry.setIndex( ( buffer.indices.length > 65535 ? THREE.Uint32Attribute : THREE.Uint16Attribute )( buffer.indices, 1 ) );
+		geometry.addAttribute( 'position', THREE.Float32Attribute( buffer.vertices, 3 ) );
+		geometry.addAttribute( 'normal', THREE.Float32Attribute( buffer.normals, 3 ) );
+		geometry.addAttribute( 'uv', THREE.Float32Attribute( buffer.uvs, 2 ) );
+		geometry.addAttribute( 'skinIndex', THREE.Float32Attribute( buffer.skinIndices, 4 ) );
+		geometry.addAttribute( 'skinWeight', THREE.Float32Attribute( buffer.skinWeights, 4 ) );
+
+		geometry.computeBoundingSphere();
+		geometry.computeFaceNormals();
```
Comment (WestLangley) [inline examples/js/loaders/MMDLoader.js]: `BufferGeometry.computeFaceNormals()` does nothing.

File: examples/js/loaders/MMDLoader.js [hunk 2]
```diff
@@ -2605,6 +2635,24 @@ THREE.MMDLoader.prototype.createMesh = function ( model, texturePath, onProgress
...
+		geometry.verticesNeedUpdate = true;
+		geometry.normalsNeedUpdate = true;
+		geometry.uvsNeedUpdate = true;
```
Comment (WestLangley) [inline examples/js/loaders/MMDLoader.js]: The `needsUpdate` flags should not be required here.

File: examples/js/loaders/MMDLoader.js (hunk 1 above)
Comment (takahirox) [inline examples/js/loaders/MMDLoader.js]: Oops, thanks!

**Comment by mrdoob** [comment]
Yay!

**Comment by mrdoob** [comment]
Thanks!

**Comment by Mugen87** [comment]
Cool! 👍

**Pull request merged**